spec/models/survey_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
```

Comment:
Ideally this file is the first commit on the PR. Run the test and watch it fail. This habit solidifies understanding of the order in which Rails looks for things. Helpful in debugging bigger issues later because you are familiar with all types of errors early on.